src/urllib3/contrib/securetransport.py

```diff
@@ -869,7 +867,7 @@ def set_alpn_protocols(self, protocols):
             raise NotImplementedError(
                 "SecureTransport supports ALPN only in macOS 10.12+"
             )
-        self._alpn_protocols = [six.ensure_binary(p) for p in protocols]
+        self._alpn_protocols = [util.util.to_bytes(p) for p in protocols]
```

Comment:
Let's expand the import to `from ..util.util import to_bytes` so it's similar to the rest of the usages

Reply:
I get this error when I import it by `from ..util.util import to_bytes`

`Could not import SecureTransport: ModuleNotFoundError("No module named 'urllib3.packages.util'")`

Reply:
`to_bytes(p, "ascii")`